staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -7093,7 +7093,7 @@ const (
 type DownwardAPIVolumeFile struct {
 	// Required: Path is  the relative path name of the file to be created. Must not be absolute or contain the '..' path. Must be utf-8 encoded. The first item of the relative path must not start with '..'
 	Path string `json:"path" protobuf:"bytes,1,opt,name=path"`
-	// Required: Selects a field of the pod: only annotations, labels, name and namespace are supported.
+	// Required: Selects a field of the pod: only annotations, labels, name, namespace and uid are supported.
```

Comment:
This is less wrong, but https://kubernetes.io/docs/concepts/workloads/pods/downward-api/ lists other fields you could select.

Reply:
If we want a short URL for supported fields, we can make one.

Reply:
That API says we support those but not in FieldRef. 

DownwardAPI supports them as a few options but we directly expose FieldRef in uuid. That comment is actually in pkg/api/core so wanted to make sure they were identical.

And for "supported fields" are you saying that we should just link to the code comment?

Reply:
Ah, yep, that's right. My mistake.